pilot/pkg/serviceregistry/serviceentry/conversion_test.go

```diff
@@ -1001,7 +1017,23 @@ func TestConvertWorkloadEntryToWorkloadInstance(t *testing.T) {
 					ServiceAccount: "scooby",
 				},
 			},
-			out: nil,
+			out: &model.WorkloadInstance{
+				Namespace: "ns1",
+				Kind:      model.WorkloadEntryKind,
+				Endpoint: &model.IstioEndpoint{
+					Labels: map[string]string{
+						"topology.istio.io/cluster": clusterID,
+					},
+					Address:        "scooby.com",
```

Comment:
Will this be sent over EDS and then fail?

Reply:
Updated to avoid selecting by non-DNS SE.